test/functions.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require '../functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase
+{
+    public function testSuccessDisplayCards_1()
+    {
+        $input = [['name' => 'England', 'place' => 'Lulworth', 'landmark' => 'Durdle Door', 'activity' => 'Walking', 'food' => 'sea food']];
+        $actualOutput = displayCards($input);
```

Comment:
without the `$expectedOutput`, `$actualOutput` sounds a bit weird so i would call it just `$output`